howto/manage/web-dashboard.md

```diff
@@ -74,3 +74,37 @@ You can select any application you previously created and start a streaming sess
 ![Start a streaming session|690x352](https://assets.ubuntu.com/v1/6087fbd9-gs_dashboard_start_session.png)
 
 For more information about the Streaming Stack, visit [About application streaming](https://discourse.ubuntu.com/t/streaming-android-applications/17769).
+
+#### Streaming statistics
+
+View the streaming statistics for your running sessions by selecting the **Statistics** button on the session. The statistics display on the right pane and also have a download option to download the statistics in a `.csv` format for further analysis.
+
+The downloaded `.csv` file has the following statistics:
+
+| Statistic | Description |
+| --------- |------------ |
+| `date` | Date and time in ISO format |
```

Comment:
Just a nitpick, but this is a bit more accurate:
```suggestion
| `date` | Date and time in ISO 8601 format |
```
You don't have to add this if you think it's not useful.

Reply:
Thanks, it is better to be accurate.